product_contract/models/sale_order.py

```diff
@@ -63,6 +66,7 @@ def _prepare_contract_value(self, contract_template):
     def action_create_contract(self):
         contract_model = self.env["contract.contract"]
         contracts = []
+        lines_without_contract = self.env["sale.order.line"]
```

Comment:
```suggestion
        lines_without_contract = self.env["sale.order.line"].browse()
```
is more correct syntaxically

Reply:
done